framework/devicedeps/cpu/cpu_device.cpp

```diff
@@ -13,6 +13,17 @@
 
 uint64_t cpu_features = 0;
 
+/// Windows cross compilation doesn't handle __attribute__((weak)) well.
+#ifdef _WIN32
+__attribute__((noclone, noinline)) void device_specific_init()
```

Comment:
The problem now is I don't see how this can work. Won't we have two symbols `device_specific_init` when another .cpp source provides this?

Reply:
Yes, you are right, it would create two definitions of `device_specific_init`. Since mingw doesn't seem to support weak symbols, I restored original solution where `device_specific_init` is defined as weak, but in the same translation unit where it is used (`sandstone.cpp`).

Reply:
How was this working before, then? Ping me tomorrow early so you can show me the errors you're getting.